Firmware/MVP/LTC68042.cpp

```diff
@@ -172,19 +178,46 @@ uint8_t LTC6804_getStackVoltage()
 
   uint8_t stackVoltage = uint8_t(stackVoltage_RAW * 0.0001);
 
+
+  // Natalya's edits below: average stackVoltage over 8 readings to smooth them out
+  uint8_t sendStackVoltage = lastSentStackVoltage;
+
+  stackVoltageSum += stackVoltage;
+
+  if (stackVoltageReadIndex > 7) {
+    // First iteration
+    lastSentStackVoltage = stackVoltage;
+    sendStackVoltage = stackVoltage;
+    stackVoltageSum = 0;
+    stackVoltageReadIndex = 0;
+  } else if (stackVoltageReadIndex == 7) {
+    stackVoltageReadIndex = -1;
+    sendStackVoltage = stackVoltageSum * 0.125;
+    stackVoltageSum = 0;
+    lastSentStackVoltage = sendStackVoltage;
+  } else sendStackVoltage = lastSentStackVoltage;
+
+  stackVoltageReadIndex += 1;
+
+  uint8_t dispStackVoltage = (uint8_t)(sendStackVoltage*0.94);
+
+  uint8_t stackVoltageMultiplied = stackVoltage*0.94;
+
```

Comment:
Correct me if I missed it, but are you using "stackVoltageMultiplied" anywhere else?